packages/runtime-core/src/renderer.ts

```diff
@@ -484,6 +484,8 @@ function baseCreateRenderer(
     // set ref
     if (ref != null && parentComponent) {
       setRef(ref, n1 && n1.ref, parentSuspense, n2 || n1, !n2)
+    } else if (ref == null && n1 && n1.ref != null) {
+      setRef(n1.ref, n1.ref, parentSuspense, n2 || n1, true)
```

Comment:
```suggestion
      setRef(n1.ref, null, parentSuspense, n1, true)
```
- `oldRawRef` can be set to `null` during unset ref
- `vnode` should be n1

Reply:
done~